content/en/docs/tasks/security/citadel-config/auth-sds/index.md

```diff
@@ -49,8 +49,7 @@ This approach has the following benefits:
 
 ## Before you begin
 
-* Follow the [install instructions](/docs/setup/install/istioctl/)
-  to set up Istio with SDS and global mutual TLS enabled.
+Follow the [Istio installation guide](/docs/setup/install/helm/) to set up Istio with SDS and global mutual TLS enabled.
```

Comment:
Can't we do this with istioctl?

Reply:
Yes, I need to test it. I will update to istioctl once I verified it's working.

Reply:
It's still not working. We will update this when we have fixed it.